src/Cake.Web/App_Data/docs.xml

```diff
@@ -35,7 +35,7 @@
     <item title="Visual Studio Code" file="editors/vscode.md" />
   </section>
   <section id="build-systems" title="Build Systems">
-    <item title="Visual Studio Online" file="build-systems/vso.md" hidden="true"/>
```

Comment:
I don't think it is a big deal.  I was just being pedantic, and thought I would change it while I was there.  I am happy to leave it as is.